packages/react-devtools-core/package.json

```diff
@@ -18,7 +18,9 @@
   "scripts": {
     "build": "yarn build:backend && yarn build:standalone",
     "build:backend": "cross-env NODE_ENV=production webpack --config webpack.backend.js",
+    "build:backend:crx": "cross-env NODE_ENV=production FEATURE_FLAG_TARGET=core/backend-fb webpack --config webpack.backend.js",
     "build:standalone": "cross-env NODE_ENV=production webpack --config webpack.standalone.js",
+    "build:standalone:crx": "cross-env NODE_ENV=production FEATURE_FLAG_TARGET=core/standalone-fb webpack --config webpack.standalone.js",
```

Comment:
Can we replace "crx" with "fb" or "internal"?

The "crx" name kind of made sense in the extension build because it's building a `*.crx` file extension (the Chrome Extension file format) to be installed by Chef. We only build a `*.crx` file for internal use, because when we submit an extension to the Chrome store we submit a `*.zip` bundle.

Maybe we should just change the other target too, to be clearer.